File: code/1012/1012.rb
line = gets.split(' ')

PI = 3.14159

A = line[0].to_f
B = line[1].to_f
C = line[2].to_f

puts format('TRIANGULO: %.3f', (A * C / 2.0))
puts format('CIRCULO: %.3f', PI * (C * C))
puts format('TRAPEZIO: %.3f', ((A + B) * C) / 2.0)
puts format('QUADRADO: %.3f', B * B)
puts format('RETANGULO: %.3f', A * B)

File: code/1012/1012.js
const input = require("fs").readFileSync("/dev/stdin", "utf8");
const lines = input.split(/\n|\s/);

const PI = 3.14159;

const A = parseFloat(lines[0]);
const B = parseFloat(lines[1]);
const C = parseFloat(lines[2]);

console.log(`TRIANGULO: ${((A * C) / 2.0).toFixed(3)}`);
console.log(`CIRCULO: ${(PI * (C * C)).toFixed(3)}`);
console.log(`TRAPEZIO: ${(((A + B) * C) / 2.0).toFixed(3)}`);
console.log(`QUADRADO: ${(B * B).toFixed(3)}`);
console.log(`RETANGULO: ${(A * B).toFixed(3)}`);
